resources/js/Components/Table/RatingTable/RatingTable.tsx

```diff
@@ -0,0 +1,86 @@
+import DoneIcon from '@mui/icons-material/Done'
+import {Box, Chip, Typography} from '@mui/material'
+import {DataTable} from 'Components'
+import React from 'react'
+import {ReportedRatingData} from 'Types/reported-rating-data'
+
+type RatingTablePropType = {
+    ratingFilterStatus: boolean
+    ratingFilterSetter: () => void
+    reportedRatings: ReportedRatingData[]
+}
+
+const RatingTable = ({
+    ratingFilterStatus,
+    ratingFilterSetter,
+    reportedRatings,
+}: RatingTablePropType) => {
+    const leftActions = [
+        <Chip
+            label='League Event Ratings Only'
+            onClick={() => ratingFilterSetter()}
+            deleteIcon={ratingFilterStatus ? <DoneIcon /> : undefined}
+            onDelete={ratingFilterStatus ? () => {} : undefined}
```

Comment:
`onDelete` here just does nothing?

Reply:
I tested this out. I believe that MUI does not show the icon next to the chip if there is no `onDelete` prop being passed. It's weird but it works.